src/cadenza.js

```diff
@@ -189,6 +192,8 @@ export class CadenzaClient {
    * @param {MediaType} [options.mediaType] - Set to 'application/pdf' for views of type "JasperReports report"
    *     to show the report PDF directly, without any Cadenza headers or footers.
    * @param {OperationMode} [options.operationMode] - The mode in which a workbook should be operated
+   * @param {FilterVariables} [options.filter] - Filter variables. The keys should contain the
```

Comment:
Please maintain the alphabetical order (in the whole file).

Reply:
Fixed.